application-blog-ui/src/main/resources/Blog/Translations.pl.xml

```diff
@@ -39,99 +39,99 @@
   <minorEdit>false</minorEdit>
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
-  <content>### Missing: blog.application.name=Blog
-### Missing: blog.archive.paneltitle=Blog Archive
-### Missing: blog.archive.pagetitle=Archive
-### Missing: blog.archive.noarticle=No articles yet...
+  <content>blog.application.name=Blog
+blog.archive.paneltitle=Archiwum bloga
+blog.archive.pagetitle=Archiwum
+blog.archive.noarticle=Brak obiektów...
 ### Missing: blog.archive.postsyear=posts for {0}
-### Missing: blog.archive.unpublished=(unpublished)
-### Missing: blog.archive.hidden=(hidden)
-### Missing: blog.archive.noarticlesyear=No articles in this year...
+blog.archive.unpublished=(nie opublikowano)
```

Comment:
The parenthesis don't look that good. WDYT?

Reply:
I'll still merge it because it's better than before. Don't know how we should handle that though.